submissions/dummy_submission/the-nuclear-options-v1.0.md

```diff
@@ -0,0 +1,233 @@
+
+ Now that we' seen how nuclear power is not so dangerous in the vanilla game, let's look over the other common concerns about it.
+
+## **Common Concerns with Nuclear Power**
+
+* **Nuclear power is too expensive to set up until the late game** – In terms of research, you do not need more than chemical science packs to unlock nuclear power technology. The setting up requires time, yes. And a single nuclear reactor sure is expensive. However the material costs, in terms of iron and copper, for a 40MW starter level nuclear setup is only [a fraction]()<!--[See appendix]--> of the cost of setting up 40MW worth of solar panels and accumulators, and the cost per MW of adding any further reactors is substantially cheaper thanks to already-established infrastructure and the substantial gains from the neighbor bonus. 
+* **A nuclear reactor is too expensive to maintain without the Kovarex enrichment process** – Indeed, obtaining one unit of Uranium-235 requires centrifuging thousands of uranium ores, but on the other hand 1 piece of it is enough to make 10 uranium fuel cells, which last over 30 minutes in total. In a rather eye-opening [video](https://youtu.be/Qw_NzPuccxk?t=1610) Nilaus walks us through the mathematics and from there we see that you need only 8-9 miners and 1 centrifuge to maintain 1 nuclear reactor continuously without the Kovarex enrichment process, although it does not hurt to process a little more in case of bad luck during centrifuging.
+
+* **Nuclear calculations and designs are hard **– At the end of the day, calculating ratios for nuclear setups is not that different from planning your assembly lines. In the same spirit, the wonderful [Factorio Cheat Sheet](https://factoriocheatsheet.com/#nuclear-power) provides the formulas and interactive calculators you need and there are just a few design rules to keep in mind, which we cover below.
+* **Nuclear plants are bad for the game performance (UPS)** – It’s true that with all the fluid mechanics involved, nuclear cannot compete with the simplicity of solar power in terms of UPS efficiency, but that does not mean its impact is that large, especially with just a handful of reactors.  We will cover this too, below.
+* **What if the radiation causes mutated mega-biters?** - Oddly enough radiation does not seem to affect any entities in vanilla Factorio while regular old pollution already is enough to mutate those pesky biters. Nuclear energy is actually very helpful in this regard, as creates no pollution apart from the relatively small amounts from uranium mining and processing.

```

Comment:
I kinda agree with Firerazor on this last point where it seems a little like reaching for concerns. You've already made quite a few very good points already. This is just a thought though, if you want to keep it - by all means. We're only here to help :)